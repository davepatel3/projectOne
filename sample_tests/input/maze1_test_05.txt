8
14
4
900137439
print_walls
0
print_walls
1
print_walls
2
print_walls
3
quit
