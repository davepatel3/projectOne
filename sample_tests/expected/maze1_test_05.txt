+---+---+---+---+---+---+---+---+
| X | X   X   X   F | F | F | F |
+   +---+---+---+---+   +   +---+
| X   X   F | F | F | X | X   X |
+---+---+---+   +   +   +---+   +
| X | F | X   X | X   X | F   X |
+   +   +---+   +---+---+---+---+
| X | F | X   X | X   X | X   X |
+   +---+   +---+   +   +   +   +
| X   F | F | X   X | X | X | X |
+   +---+---+   +---+   +   +   +
| X   F | F | F | X | X | X | X |
+---+---+   +---+   +   +   +   +
| X | F   X | F | X | X | X | F |
+   +---+---+   +   +   +   +---+
| F | F   X   X | F | F | X | F |
+---+---+---+---+   +---+   +---+
| X   F | F   X   F   X | X   X |
+   +---+---+   +---+   +---+   +
| X | F   X | X | F | X   X | F |
+   +---+   +---+   +---+   +---+
| X   X | F | F | X   F | X   X |
+   +---+---+   +---+---+---+   +
| F | X   X | F | F | X   F | F |
+   +   +   +---+---+   +---+---+
| X | X | X   X   X | X | F   X |
+   +   +---+---+   +   +---+   +
| X | F | F   F | X   X | X   F |
+---+---+---+---+---+---+---+---+

+---+---+---+---+---+---+---+---+
| F | F   F | F |     B | B   B |
+   +---+---+   +---+---+---+   +
| X | F | B | B | B   X   X   X |
+   +---+   +---+---+---+---+   +
| F | B   X | X   F | F | B | X |
+---+---+---+   +---+---+   +   +
| F | B   F | X   F | X   X | X |
+   +---+---+---+---+   +---+---+
| F |   | B   X   X | X   X   X |
+---+---+---+---+   +---+---+   +
| X   B | B |   | X | X   X   X |
+   +---+   +---+   +   +---+---+
| X   B | X | B   X | X   X   B |
+---+---+   +---+---+---+---+---+
| B |   | F | F   B |   | X   B |
+   +---+---+---+---+---+   +   +
| X |     B | F |   | F   X | X |
+   +---+   +   +---+---+   +---+
| X |   | X | F | B   F | F | B |
+   +---+---+---+---+---+---+   +
| F | X   B | B   X   B | F | X |
+   +   +---+---+---+---+   +   +
| B | F | F |   |     F | B | B |
+---+---+---+---+---+   +---+---+
| F | F | F | X   X | X | B | X |
+   +---+   +   +   +   +   +   +
| F |   | B | B | F | F | F | B |
+---+---+---+---+---+---+---+---+

+---+---+---+---+---+---+---+---+
|   |   |   | B |   | X   F   F |
+---+---+---+   +---+   +---+   +
| F |     X | F | F | F | F | X |
+---+---+   +---+---+---+   +---+
| B   F | F | F   B |   | X | F |
+---+---+---+---+---+---+   +   +
| B   X   B | X | B   X | X | X |
+---+---+---+   +---+   +   +   +
| B | B | X   X   F | F | F | F |
+   +   +   +---+---+---+---+---+
| F | X | X | B   X | F | X   X |
+---+   +---+---+   +   +   +   +
| F | X   X   X   X | X | F | X |
+   +---+---+---+---+   +---+   +
| X |   | B | B   X | B | X   X |
+   +---+   +---+---+---+   +---+
| X |   | F |   | B | B | X   X |
+   +---+---+---+   +---+---+   +
| F | B   X |   | F |   |   | X |
+---+---+   +---+---+---+---+   +
| B   X | X   X | F   X   B | F |
+---+---+---+   +---+---+---+---+
| F   B |   | B | B | B | F   X |
+---+---+---+---+   +---+---+   +
|   |   | B | F | X   X   X   X |
+---+---+   +   +---+---+---+---+
|   |   | X   X |   |   | B   F |
+---+---+---+---+---+---+---+---+

+---+---+---+---+---+---+---+---+
| B   B | B | X   B   X   B | B |
+---+---+   +---+---+---+---+---+
| B   B | X   B | B   B | B   X |
+   +---+---+---+   +---+---+   +
| X | B | B | B | X   B   X | B |
+   +   +   +   +   +---+---+---+
| X | X   X | X | X | X   X | X |
+   +---+---+   +   +   +   +   +
| X   X   X   X | B | B | B | B |
+---+---+---+---+   +---+---+   +
| B   X   X   X | X | B   X   X |
+---+---+---+   +   +---+---+---+
| B | X   X | X | X   X   B | X |
+   +---+   +   +---+---+---+   +
| X   B | X | X   X | X   X   X |
+---+---+   +---+   +---+---+   +
| X   B | B   B | X   X | X   X |
+---+   +---+---+---+   +   +   +
| B | X   X | B   B | B | B | X |
+   +---+---+---+---+---+   +---+
| X   X   X   X   B | X   X   B |
+---+---+---+---+---+   +---+---+
| B | X   B   X   X | X | B | X |
+   +   +   +---+   +   +   +   +
| B | B | X | B | X | X   X | X |
+---+---+   +   +---+---+---+   +
| B   B | X   X | B   B   X   B |
+---+---+---+---+---+---+---+---+

